filepath_test.go

```diff
@@ -94,6 +96,37 @@ func TestLineCntByReader(t *testing.T) {
 	assert.Equal(t, 20, LineCntr(file))
 }
 
+func TestLineCntByReader_error(t *testing.T) {
+
+	var r io.Reader
+
+	r = mockReader{}
```

Comment:
@adelowo This PR is not gonna be merged. Due to real world usage performance impact. However I would like mention it here, I saw similar in your request access log implementation. 

Code Guidelines:
* Please do not added un necessary newline in the code. Added only when logical group of lines
* Test case name always Pascal case, please don't mix Pascal and Snake case together
* Use `goreturns` to format the source code.

For e.g.:
```go
func TestLineCntByReaderError(t *testing.T) {
	// implementation
	var r io.Reader
	r = mockReader{}
	defer CloseQuietly(r)

	// assertion is logical group
	assert.Equal(t, -1, LineCntr(r))
}
```